src/clojure/nrepl/middleware/caught.clj

```diff
@@ -0,0 +1,31 @@
+(ns nrepl.middleware.caught
```

Comment:
I’m not vary fond of the term `caught`, as least not without some qualifier. Something with error/exception in its name would probably convey more sense.

Reply:
Or, as I just realize, we can make this a generic middleware for removing values from the responses and then we’d need a better name. :-)